test/io.jl/save_resume_chain.jl

```diff
@@ -19,33 +19,39 @@ alg3 = Gibbs(500, PG(30, 10, :s), HMCDA(1, 500, 0.65, 0.05, :m))
 end
 
 chn1 = sample(gdemo([1.5, 2.0]), alg1; save_state=true)
-
 check_numerical(chn1, [:s, :m], [49/24, 7/6])
 
-chn1_resumed = resume(chn1, 1000)
+# Test chains read/write.
+write("chn.jlds", chn1)
```

Comment:
Maybe we should move this IO test to `MCMCChains?`

Reply:
Works for me, I'll pull it out.